### DISCUSSION TITLE
Cách tái truyền động dữ liệu vào component và tái sử dụng nó

### DISCUSSION THREAD

**Discussion by hai-duc** [OngDev/community#174] [Q&A]
Hello ae, mình newbie cần ae sp cho phần tái sử dụng component. Ví dụ như là 1 component form muốn thay đổi động các trường dữ liệu trong từng page khác nhau thì làm ntn.

**Comment by popmusicfan1992** [comment]
Mình nghĩ bạn nên tìm hiểu HOC ( higher order component)
